site/docs/foundations/duration.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Duration
+layout: DetailTechnical
+sidebar:
+  label: Duration
+data:
+  status: In progress
+---
+
+The time between the moment a user performs an action and the moment feedback is given by the system. Consistency in timing provides comfort and can increase trust in a system.
+
+## Duration scale
+
+There are four durations available to choose from; Instant, Perceptible, Notable, and Cut-off.
+
+| Token                  | Duration (milliseconds) |
+| ---------------------- | ----------------------- |
+| `duration-instant`     | 100                     |
```

Comment:
the token in code are actually written `--salt-duration-insant` which we should be clear about. I know they are represented different in Figma and we had an idea to toggle between showing the code in figma vs code which would be great. but I would opt for having a matching token for one of the two for now, until we can add the other, or otherwise even add in two columns, one for each?

Reply:
@libertymayc thanks. I’m OOO at the moment but once I’m back I’ll make the change across all foundations.